test/unicode-spec.js

```diff
@@ -9,7 +9,7 @@ import {strictSettings, nonstrictSettings} from "./helpers";
 describe("unicode", function() {
```

Comment:
The titles for these tests should change from `it("should parse...` to `it("should build...`.